api/envoy/api/v2/route/route.proto

```diff
@@ -180,6 +180,9 @@ message Route {
 
     // Return an arbitrary HTTP response directly, without proxying.
     DirectResponseAction direct_response = 7;
+
+    // Return an noop action
```

Comment:
What does returning a noop action mean? Rephrase?

Reply:
This was part of my previous feedback, I would prefer this to be called `Fallthru` or something like that. @pitiwari can you address my earlier feedback and I'll take another pass, thanks.